src/vs/workbench/test/browser/workbenchTestServices.ts

```diff
@@ -1583,6 +1583,7 @@ export class TestTerminalInstanceService implements ITerminalInstanceService {
 	async getXtermUnicode11Constructor(): Promise<any> { throw new Error('Method not implemented.'); }
 	async getXtermWebglConstructor(): Promise<any> { throw new Error('Method not implemented.'); }
 	createWindowsShellHelper(shellProcessId: number, xterm: any): any { throw new Error('Method not implemented.'); }
```

Comment:
I think this and `getDefaultShellAndArgs`/`getMainProcessParentEnv` can be removed